smt/pkg/blockinfo/keys.go

```diff
@@ -0,0 +1,65 @@
+package blockinfo
+
+import (
+	"errors"
+	"github.com/ledgerwatch/erigon/smt/pkg/utils"
+	"math/big"
+)
+
+const IndexBlockHeaderParamBlockHash = 0
+const IndexBlockHeaderParamCoinbase = 1
+const IndexBlockHeaderParamNumber = 2
+const IndexBlockHeaderParamGasLimit = 3
+const IndexBlockHeaderParamTimestamp = 4
+const IndexBlockHeaderParamGer = 5
+const IndexBlockHeaderParamBlockHashL1 = 6
+const IndexBlockHeaderParamGasUsed = 7
+
+const IndexBlockHeaderParam = 7
```

Comment:
why is this also 7 and what does this one mean?

Reply:
JS implementation copy pasta, fixed now